### DISCUSSION TITLE
How to contribute translation?

### DISCUSSION THREAD

**Discussion by xialeistudio** [linkerd/linkerd2#8595] [Q&A] [answered]
I think the Linkerd is a good project.
How to contribution translation?

**Comment by mateiidavid** [comment, marked as answer]
Hi @xialeistudio, thanks for your interest in contributing!

At the moment, we support translations for the dashboard page. If you'd like to contribute translations, I'd look at [the locales directory in the web dashboard](https://github.com/linkerd/linkerd2/tree/main/web/app/js/locales). There's a Spanish and English translation.

I'm not super familiar with how this works, but I'm more than happy to guide where possible. Have a look at our [BUILD.md](https://github.com/linkerd/linkerd2/blob/main/BUILD.md#translations) file, it contains instructions on how to add translations. Here's a PR on how `ES` translations were done: https://github.com/linkerd/linkerd2/pull/4866

**Reply by xialeistudio** [reply]
Thanks for your reply!
I'll try it.
